rero_ils/modules/contributions/mappings/v7/contributions/contribution-v0.0.1.json

```diff
@@ -359,6 +359,9 @@
       "organisations": {
         "type": "keyword"
       },
+      "deleted": {
```

Comment:
```suggestion
      "deletion_date": {
```

Reply:
If I change it, I have to change it in rero-mef too. There other date fields named `_updated` and `_created`.